WBooks/Repositories/BookRepository.swift

```diff
@@ -28,8 +30,49 @@ class BookRepository {
                 }
             }
     }
+    
+    public func rentBook(bookId: Int, onSuccess: @escaping () -> Void, onError: @escaping (Error) -> Void) {
```

Comment:
This is a User request so you could define this in a different repository `UserRepository`